cypress/e2e/Album.cy.js

```diff
@@ -0,0 +1,80 @@
+describe('AlbumDetails', () => {
+  before(() => {
+    cy.intercept(
+      'GET',
+      'https://api.discogs.com/masters/25976?key=GimREdkHlKcSjALMSwEP&secret=RZbpExNDRyTdbTAaiVxiJpiYgOcydrMJ',
+      {
+        statusCode: 200,
+        body: {
+          "title": "Clicked Album Title",
+          "artist": "Clicked Artist",
+          "releaseDate": "2023-07-01",
+          "genre": "Pop",
+          "styles": ["Style1", "Style2"],
+          "tracklist": ["Track 1", "Track 2", "Track 3"],
+          "coverImg": "https://example.com/clicked_album_cover.jpg"
+        }
+      }
+    ).as('getClickedAlbum');
+    cy.visit('localhost:3000');
+  });
+
+  it('As a user, I should be able to click on a specific album and see its details', () => {
+    cy.intercept(
+      'GET',
+      'https://api.discogs.com/database/search?q=outkast&type=master&key=mbubAaAXseWPUpaJLkKU&secret=TrELhUezCNdFoIfmoAdHZmfJIXljOSfW&format=vinyl&page=1',
+      { fixture: 'results.json' }
+    ).as('searchResults');
+
+    cy.intercept(
+      'GET',
+      'https://api.discogs.com/database/search?type=master&format=vinyl&key=GimREdkHlKcSjALMSwEP&secret=RZbpExNDRyTdbTAaiVxiJpiYgOcydrMJ&page=1&per_page=5&sort=hot',
+      {}
+
+    );
+
+    cy.get('.search--form').should('be.visible')
+      .get('.search--input').type('outkast')
+      .get('.search--button').click()
+      .get('.results').should('be.visible');
+  
+    cy.get('#25976 > .results--image').first().click();
```

Comment:
This is actually breaking because you didn't provide a fixture to the intercept
<img width="814" alt="Screenshot 2023-07-30 at 10 31 00 AM" src="https://github.com/mbenfowler/sound-stash/assets/32660020/feab145c-2ea5-4d39-a89a-59b6069c5274">

Reply:
"Still have some Cypress testing that i have left on this Component. Ive been trying to figure out why my specific album fixture is not Stubbing correctly and populating in the Cypress testing"

Reply:
Again testing. But I opened another ticket to deal with info when you click from the collections page. Anything saved to collections does not display the correct info.